codalab/lib/bundle_cli.py

```diff
@@ -2339,37 +2339,31 @@ def do_print_command(self, args):
     def display_interpreted(self, client, worksheet_info, interpreted):
         for item in interpreted['items']:
             mode = item['mode']
-            data = item['interpreted']
-            properties = item['properties']
             print >>self.stdout, ''  # Separate interpreted items
-            if mode == 'markup' or mode == 'contents':
-                if mode == 'contents':
-                    maxlines = properties.get('maxlines')
+            if mode == 'markup_block' or mode == 'contents_block':
+                bundle_info = item['bundles_spec']['bundle_infos'][0]
```

Comment:
If markup, this shouldn't exist, right?

Reply:
Probably these two modes should just be separate branches of the if.

Reply:
yeah, will definitely change this.